gotechnica/platform-2020
# fill in user info on registration page

### Description
Make a call to the DB when the registration page is loaded to prefill values already loaded in for the user into the text boxes

addressed ticket: https://trello.com/c/hqizERGO/507-users-as-a-hacker-i-want-my-information-to-be-pre-filled-when-registering-if-it-already-exists-in-the-database

### Steps to Test
Create user with any information you want. Then, go to http://localhost:8080/register and see the information is prefilled in
